kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/DerivedResourcesTest.java

```diff
@@ -88,6 +89,23 @@ public static List<VirtualKafkaCluster> virtualKafkaClustersFromFiles(Set<Path>
         return clusters;
     }
 
+    public static List<KafkaProxyIngress> kafkaProxyIngressesFromFiles(Set<Path> paths) {
+        // TODO should validate against the CRD schema, because the DependentResource
+        // should never see an invalid resource in production
```

Comment:
It's true that we want to avoid accidentally using schema-invalid resources in our tests. However, the validation that the api server does is more than just JSONSchema validation (because Kube CRDs `openApiv3Schema` is not exactly the same as the real open API v3). So we must choose our poison: no validation, possibly incorrect validation, or slow minikube/kind based real validation.